src/tests/unit/tests/common/message-creators/issue-filing-action-message-creator.test.ts

```diff
@@ -77,36 +76,23 @@ describe('IssueFilingActionMessageCreator', () => {
         const telemetry = { ...telemetryStub, service: testService };
 
         telemetryFactoryMock.setup(factory => factory.forFileIssueClick(eventStub, source, testService)).returns(() => telemetry);
-        const issueDetailsData: CreateIssueDetailsTextData = {
-            pageTitle: 'pageTitle<x>',
-            pageUrl: 'pageUrl',
-            ruleResult: {
-                failureSummary: 'RR-failureSummary',
-                guidanceLinks: [{ text: 'WCAG-1.4.1' }, { text: 'wcag-2.8.2' }],
-                help: 'RR-help',
-                html: 'RR-html',
-                ruleId: 'RR-rule-id',
-                helpUrl: 'RR-help-url',
-                selector: 'RR-selector<x>',
-                snippet: 'RR-snippet   space',
-            } as DecoratedAxeNodeResult,
-        };
-
-        dispatcherMock.setup(dispatcher =>
-            dispatcher.dispatchMessage(
-                It.isValue({
-                    messageType: Messages.IssueFiling.FileIssue,
-                    payload: {
-                        service: testService,
-                        issueData: issueDetailsData,
-                        telemetry,
-                    },
-                }),
-            ),
-        );
+        const issueDetailsData: CreateIssueDetailsTextData = {} as any;
```

Comment:
Just wondering, why are we casting `issueDetailsData as any` here but using `issueDetailsData: {} as CreateIssueDetailsTextData` in issue-filing-button.test?

Reply:
I think both are similar & don't know of any meaningful differences - afaik it's preference / what nearby code has already that might determine what one chooses